<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="1.18s"
		durationms="1180"
		type="root"
		rid="83b713fb3c940c0"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="I-Ray"
		rxversion="10.5.1.0"
		host="IRAYSERVER"
		osversion="Windows 10 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="en-US"
		dtnlanguage="en-US"
		screenresolution="1600x900"
		timestamp="10/20/2023 12:47:06 PM"
		timestampiso="2023-10-20T07:17:06"
		endtime="10/20/2023 12:47:07 PM"
		timeoutfactor="1"
		runid=""
		sutversion="">
		<detail>
			Ranorex Testscript
		</detail>
		<item
			timeRelativeToTestSuiteStartTime="00:01.173"
			timeRelativeToTestModuleStartTime="00:01.173"
			timeWallClock="12:47:07"
			time="00:01.173"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
